README.adoc

```diff
@@ -2198,6 +2198,19 @@ something = something && something.downcase
 something &&= something.downcase
 ----
 
+=== Equal Object Identifier [[equal-object-id]]
+
+Prefer `equal?` over `==` when comparing `object_id`. `Object#equal?` method is provided to compare objects for identity, and comparing objects for equality using `Object#==` does not require redundant `object_id` to be repeated.
```

Comment:
> comparing objects for equality using `Object#==` does not require redundant `object_id` to be repeated.

I think this part can be worded better, or changed to something like "and in contrast `Object#==` is provided for the purpose of doing value comparison" or something along those lines.